mubench.reviewsite/src/Controllers/ReviewsController.php

```diff
@@ -131,7 +139,55 @@ public function updateOrCreateReview($misuse_id, $reviewer_id, $comment, $findin
             $findingReview = FindingReview::firstOrNew(['review_id' => $review->id, 'rank' => $rank]);
             $findingReview->decision = $findings_review['hit'];
             $findingReview->save();
-            $findingReview->violation_types()->sync($findings_review['types']);
+            if($findings_review['types']){
```

Comment:
Why is this necessary and what does it have to do with the navigation feature? Doesn't this break the case where I remove all types from a misuse?